lib/taurus/core/tango/tangoattribute.py

```diff
@@ -782,13 +789,18 @@ def push_event(self, event):
             if etype is None:
                 return
             manager = Manager()
-            sm = self.getSerializationMode()
             listeners = tuple(self._listeners)
-            if sm == TaurusSerializationMode.Concurrent:
-                manager.addJob(self.fireEvent, None, etype, evalue,
-                               listeners=listeners)
-            else:
+            sm = self._serialization_mode
+            if sm == TaurusSerializationMode.TangoSerial:
+                self.deprecated(dep='TaurusSerializationMode.TangoSerial mode',
+                                alt="TaurusSerializationMode.Serial",
+                                rel='4.3.2')
                 self.fireEvent(etype, evalue, listeners=listeners)
+            else:
+                manager.enqueueJob(self.fireEvent, None,
```

Comment:
It would be better and more consistent to pass the callback arg as a kwarg (i.e., use callback=None instead of None)

Reply:
In fact I removed it, since it is not necessary and it is the default value